### PR TITLE
[ibexa/*] Synchronized changes between 3.3.x-dev and 3.3

### PR DESCRIPTION
**Pull request by webhdx**
Some changes were only applied to `3.3` or `3.3.x`:
* #14 
* #11 

This PR prepares recipes for 3.3.1 release.

### CODE REVIEW

**Review by Steveb-p — APPROVED** [review]

File: ibexa/oss/3.3.x-dev/templates/themes/standard/pagelayout.html.twig [hunk 1]
```diff
@@ -7,7 +7,7 @@
         {% set title = ez_content_name( content ) %}
     {% endif %}
     <title>{{ title|default( 'Home' ) }}</title>
-    <meta name="generator" content="eZ Platform"/>
+    <meta name="generator" content="Ibexa Platform"/>
```
Comment (alongosz) [inline ibexa/oss/3.3.x-dev/templates/themes/standard/pagelayout.html.twig]: Following the internal sync: 
```suggestion
    <meta name="generator" content="Ibexa Open Source"/>
```

File: ibexa/oss/3.3/templates/themes/standard/pagelayout.html.twig [hunk 2]
```diff
@@ -7,7 +7,7 @@
         {% set title = ez_content_name( content ) %}
     {% endif %}
     <title>{{ title|default( 'Home' ) }}</title>
-    <meta name="generator" content="Ibexa DXP"/>
+    <meta name="generator" content="Ibexa Platform"/>
```
Comment (alongosz) [inline ibexa/oss/3.3/templates/themes/standard/pagelayout.html.twig]:
```suggestion
    <meta name="generator" content="Ibexa Open Source"/>
```

**Review by alongosz — CHANGES_REQUESTED** [review]

File: ibexa/oss/3.3.x-dev/templates/themes/standard/pagelayout.html.twig (hunk 1 above)
Comment (webhdx) [reply ibexa/oss/3.3.x-dev/templates/themes/standard/pagelayout.html.twig]: Fixed

File: ibexa/oss/3.3/templates/themes/standard/pagelayout.html.twig (hunk 2 above)
Comment (webhdx) [reply ibexa/oss/3.3/templates/themes/standard/pagelayout.html.twig]: Fixed in https://github.com/ibexa/recipes/pull/15/commits/db5e249dd30e0a00d77097f8d08aa7f2736588dd

**Review by lserwatka — APPROVED** [review]

**Pull request merged**